<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VATPeriodDlg</class>
 <widget class="QDialog" name="VATPeriodDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>256</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VATPeriod dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>VAT year:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>VAT period:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="VATYearEdit">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="displayFormat">
        <string>yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="VATPeriodBox">
       <item>
        <property name="text">
         <string>Q1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q4</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>152</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="OkButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
